docs/dotnet/dotnet-signalr-metrics.md

```diff
@@ -0,0 +1,82 @@
+# Semantic Conventions for SignalR metrics emitted by ASP.NET Core
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for SignalR metrics emitted by .NET components and runtime.
+
+**Disclaimer:** These are initial .NET metric instruments available in .NET 8 but more may be added in the future.
+
+<!-- toc -->
+
+- [Metric: `signalr.http.server.connection.duration`](#metric-signalrhttpserverconnectionduration)
+- [Metric: `signalr.http.server.active_connections`](#metric-signalrhttpserveractive_connections)
+
+<!-- tocstop -->
+
+## Metric: `signalr.http.server.connection.duration`
+
+<!-- semconv metric.signalr.http.server.connection.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `signalr.http.server.connection.duration` | Histogram | `s` | The duration of connections on the server. [1] |
+
+**[1]:** Corresponding `EventCounter` name is `connections-duration`; Meter name is `Microsoft.AspNetCore.Http.Connections`
+<!-- endsemconv -->
+
+<!-- semconv metric.signalr.http.server.connection.duration(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `signalr.connection.status` | string | SignalR HTTP connection closure status. | `app_shutdown`; `timeout` | Recommended |
+| `signalr.http.transport` | string | SignalR transport - https://github.com/dotnet/aspnetcore/blob/main/src/SignalR/docs/specs/TransportProtocols.md | `web_sockets`; `long_polling` | Recommended |
```

Comment:
This attribute might end up with the unfortunate name `signalr_http_transport.transport`

Reply:
`signalr_http_transport.type` following https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.connections.httptransporttype?view=aspnetcore-7.0?

Reply:
My preference for naming is still to use `signalr.http` but I'll leave the rationale and discussion of that in the other thread. If we did that the name doesn't sound so awkward.

If we do put transport in the prefix then I like @lmolkova's suggestion.